src/interfaces/IMenu.tsx

```diff
@@ -0,0 +1,5 @@
+export default interface IMenu {
```

Comment:
Interfaces designed only to be used for that specific file for props can and should stay in that file, any other interface that could be re-used in the future should go inside this interfaces folder.